pkg/server/routes.go

```diff
@@ -138,8 +139,14 @@ func (s *Server) routes() {
 	var emailClient email.Client
 	switch {
 	case s.environment.Deployed():
+		// Parse the regex config and pass it to the Sender if configured
+		sesRegexString := s.Config.GetString(appconfig.SESRecipientRegexKey)
+		var sesRegex *regexp.Regexp
+		if sesRegexString != "" { // only attempt to parse if it's a non-empty string
+			sesRegex = regexp.MustCompile(sesRegexString)
+		}
 		sesConfig := s.NewSESConfig()
-		sesSender := appses.NewSender(sesConfig, s.environment)
+		sesSender := appses.NewSender(sesConfig, s.environment, sesRegex)
```

Comment:
The email config is shared across the `local` and `appses` packages, and I only really wanted to add this to the SES side of things, so I opted to just add it to `appses.NewSender` instead of modifying `s.NewEmailConfig`

Happy to refactor that and just have the config value ignored by the local package though, if that's preferred!

Reply:
If you meant adding it to the `NewSESConfig`, though (not `NewEmailConfig`) I could see that as a reasonable choice too!

Reply:
a30c9fab0